### PR TITLE
 AUTH-5428: added app launcher customization fields to access applica…

### PR DESCRIPTION
**Pull request by russellkerns**
corresponding cf-go PR https://github.com/cloudflare/cloudflare-go/pull/1407

### CODE REVIEW

**Comment by jacobbednarz** [comment]
@russellkerns is this still something you're interested in getting over the line? looks like after you resolve the conflicts, this should be good.

**Comment by jacobbednarz** [comment]
just investigating why the `name` property is static 

```
TF_ACC=1 go test ./internal/sdkv2provider -v -run "^TestAccCloudflareAccessApplication_WithAppLauncherCustomization" -count 1 -timeout 120m -parallel 1
=== RUN   TestAccCloudflareAccessApplication_WithAppLauncherCustomization
    resource_cloudflare_access_application_test.go:601: Step 1/1 error: Check failed: Check 2/16 error: cloudflare_access_application.adhkxhahrj: Attribute 'name' expected "adhkxhahrj", got "App Launcher"
--- FAIL: TestAccCloudflareAccessApplication_WithAppLauncherCustomization (7.58s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider	8.678s
FAIL
make: *** [testacc] Error 1
```

other than that, we're good to merge here.

File: internal/sdkv2provider/schema_cloudflare_access_application.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func resourceCloudflareAccessApplicationSchema() map[string]*schema.Schema {
 		},
 		"name": {
 			Type:        schema.TypeString,
-			Required:    true,
+			Computed:    true,
```
[the remark this answers is not in the document]
Comment (jacobbednarz) [reply internal/sdkv2provider/schema_cloudflare_access_application.go:37]: 👍 is it also `Optional`? i.e. the user may provide it and we use it?

[the remark this answers is not in the document]
Comment (russellkerns) [reply internal/sdkv2provider/schema_cloudflare_access_application.go:37]: yes, updated that

File: terraform-provider-cloudflare_99.0.0
Comment (jacobbednarz) [inline terraform-provider-cloudflare_99.0.0:1]: you'll want to remove this binary from your PR :)

**Pull request merged**